tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -5560,7 +5699,7 @@ def get_prod(a):
     assert f_bl_true_output == test_bool_true_output
     assert f_bl_false_output == test_bool_false_output
 
-    assert (type(f_bl_true_output) == type(test_bool_false_output.item()))
+    assert matching_types(f_bl_true_output, test_bool_false_output)
```

Comment:
this test should compare the same function type

Reply:
Fixed 4a03f72